internal/pkg/v2/bootstrap/container/container.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/controller/errorconcept"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+
+	"github.com/edgexfoundry/go-mod-messaging/messaging"
+)
+
+type Container struct {
+	ConfigClient       *config.ConfigurationStruct
+	DBClient           interfaces.DBClient
+	ErrorHandlerClient *errorconcept.Handler
+	EventChan          chan<- interface{}
+	LoggingClient      logger.LoggingClient
+	MessageClient      messaging.MessageClient
+	MetadataClient     metadata.DeviceClient
+}
+
+var BootstrapContainer *di.Container
+var RegistryClients Container
+
+func Init() {
+	dic := BootstrapContainer
```

Comment:
Where is` BootstrapContainer` initialized? If uses receiver approach it can be done in the New func. Also, Why the need to reassign it to `dic`

Reply:
Refactor this part with a new v2 `BootstrapHandler` separated from v1, so the above code has been removed.